tools/shared/utils/common/utils.cpp

```diff
@@ -437,7 +437,12 @@ collectIncludePaths(int argc, char *argv[])
       while (*cp && isspace(*cp)) {
         cp++;
       }
-      if (!isPathSeparator(*cp)) {
+      /* On Windows we have a drive letter before the first path separator. */
+#ifdef _WIN64
+      if (!isPathSeparator(*(cp+2))) {
```

Comment:
If users would write `-IC` then they would still get the error `Include file path must be a full path name` because there is no path separator after the first two characters.

Reply:
Sure, but it could still have a path separator because the third character is out of bounds and accessing it undefined behaviour.